src/components/entry/Review.js

```diff
@@ -146,7 +147,16 @@ const Review = ({
       {!editable && (
         <cite>
           Reviewed on {formatISOString(review.created_at)}
-          {review.author && <> by {review.author}</>}
+          {review.author && (
+            <>
+              {' by '}
+              {review.user_id ? (
```

Comment:
Not sure how to style this - maybe text-decoration: none (undoing the underline styling) and theme.orange for color?
![Screenshot from 2024-11-11 10-24-00](https://github.com/user-attachments/assets/4394883c-0e6a-4d04-ae12-aedd0fd9624d)

Reply:
Even undoing all the link styling is an option - then the feature won't be discoverable on mobile for now, and will only be known to be a link on desktop because of pointer cursor, but that's fine, since we don't have much content for the pages yet.